### DISCUSSION TITLE
Pixelated results

### DISCUSSION THREAD

**Discussion by Finn-82** [NVlabs/instant-ngp#1331] [General]
Hi.

I'm not a dev and I can't code, so I was quite excited when Instant NGP released the no-code version. I tried to set it up with the help of tutorials, and I basically got it running. But whenever I run the test sample, the result is not as crystal-clear as in the tutorials I watched. Instead, it looks kind of "heavily pixelated" and I can't figure out why. Can anybody help me find the reason?

(Technical background for reference: PC with Windows 10, 16 GB RAM, intel Core i7, x64, NVDIA GeForce GTX 1660 Super. I installed CUDA Toolkit 12.1, Visual Studio 2022 Community (C++), cmake-3.25.3 and Python 3.10.10)

![InstandNGP_pixelated](https://user-images.githubusercontent.com/131042853/232744054-7d5cde21-8eb3-466b-a19b-586b77fbe81c.png)

**Comment by ccarlsonpb** [comment]
The pixellation you're seeing is because DLSS is turned off (because it's not supported on your 1660). I'm not sure if there is another way to get clearer results, but for me enabling DLSS makes it go from pixellated to smooth.

**Reply by Finn-82** [reply]
Aaah! That makes sense. Thanx!

**Comment by Tom94** [comment]
You can also uncheck the "dynamic resolution" checkbox, which will bring your framerate down but render at 1080p.

**Comment by Timothy-pop** [comment]
"Oh, I see. In that case, does enabling DLSS affect the final result? For example, if I get an ingp snapshot from training without DLSS and then move it to a computer that supports DLSS, will turning it on make the effect equivalent to training directly on a computer with DLSS support?

I have the no-code version installed directly on my Windows system. On Ubuntu, I deployed Instant-ngp, but it seems like it doesn't support DLSS, and I'm not sure why. My graphics card is an RTX-4090."
